modules/tinymce/src/plugins/image/test/ts/browser/plugin/MainTabTest.ts

```diff
@@ -47,4 +48,31 @@ describe('browser.tinymce.plugins.image.plugin.MainTabTest', () => {
       '</figure>'
     ));
   });
+
+  it('TINY-6400: render image_class_list and image_caption as Dialog type "group"', async () => {
+    const editor = hook.editor();
+    editor.execCommand('mceImage');
+    const dialog = await TinyUiActions.pWaitForDialog(editor);
+    const expected = {
+      ['.tox-form__grid--2col button[aria-label="Class"]']: 1,
+      ['.tox-form__grid--2col .tox-checkbox']: 1
+    };
+
+    Assertions.assertPresence('Does have columns for the class', expected, dialog);
+    TinyUiActions.clickOnUi(editor, 'button.tox-button:contains(Cancel)');
+  });
+
+  it('TINY-6400: render image_class_list as Dialog type "panel"', async () => {
```

Comment:
Can you please also add a test for when `image_caption` is true and `image_class_list` is not defined?

- [ ] Add addition test case for the issue I mentioned needed to be fixed in my previous review